### PR TITLE
New vacuum/analyze argument format.

### PR DESCRIPTION
**Pull request by bstein-clever**
**JIRA**:

**Overview**:

**Testing**:

**Roll Out**:
- [ ] This repo will auto deploy after you merge. To do a manual deploy, use ark start -e production s3-to-redshift or ark start -e production s3-to-redshift-fast

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -216,16 +216,16 @@ func runCopy(
...
+		vacuumArgs := map[string]string{
+			"tables":      inputConf.Schema + `."` + inputTable.Name + `"`,
+			"vacuum_mode": "delete",
+		}
 		// If we truncated, run an analyze as well
 		if truncate {
-			payload, err = json.Marshal(map[string]string{
-				"analyze": inputConf.Schema + `."` + inputTable.Name + `"`,
-			})
+			vacuumArgs["analzye_mode"] = "full"
```
Comment (jansenclever) [inline main.go]: typo `analzye`

File: main.go [hunk 2]
```diff
@@ -216,16 +216,16 @@ func runCopy(
...
 		}
 
+		payload, err := json.Marshal(vacuumArgs)
```
Comment (jansenclever) [inline main.go]: Does this kick off an individual worker or a whole workflow?

File: main.go (hunk 1 above)
Comment (bstein-clever) [reply main.go]: Oof, good catch, thanks. My keyboard's been acting up lately. Probably low batteries. :(

File: main.go (hunk 2 above)
Comment (bstein-clever) [reply main.go]: It should be the entire workflow, but I will monitor and confirm.

**Pull request merged**